pyomo/common/tests/test_tee.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright 2017 National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+
+import os
+import time
+
+from six import StringIO, BytesIO
+
+from pyomo.common.log import LoggingIntercept
+import pyomo.common.unittest as unittest
+
+import pyomo.common.tee as tee
+
+class TestTeeStream(unittest.TestCase):
+    def test_stdout(self):
+        a = StringIO()
+        b = StringIO()
+        with tee.TeeStream(a,b) as t:
+            t.STDOUT.write("Hello\n")
+        self.assertEqual(a.getvalue(), "Hello\n")
+        self.assertEqual(b.getvalue(), "Hello\n")
+
+    def test_err_and_out_are_different(self):
+        with tee.TeeStream() as t:
+            out = t.STDOUT
+            self.assertIs(out, t.STDOUT)
+            err = t.STDERR
+            self.assertIs(err, t.STDERR)
+            self.assertIsNot(out, err)
+
+    @unittest.skipIf(not tee._peek_available,
+                     "This test is not deterministic if select/peek "
+                     "are available")
```

Comment:
Should this be "if select/peek are **not** available"?

Reply:
Yes.  Fixed.